modules/installation-user-infra-machines-iso.adoc

```diff
@@ -68,14 +68,15 @@ $ curl -k http://<HTTP_server>/bootstrap.ign <1>
 Replace `bootstrap.ign` with `master.ign` or `worker.ign` in the command to validate that the Ignition config files for the control plane and compute nodes are also available.
 
 ifndef::openshift-origin[]
-. Obtain the {op-system} images that are required for your preferred method of installing operating system instances from the
+. Obtain the {op-system} images that are required for your preferred method of installing operating system instances for your target cluster architecture from the
 ifndef::ibm-power[]
-link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.10/[{op-system} image mirror]
+link:https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.10/[{op-system} (x86_64) image mirror] or
+link:https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rhcos/4.10/[{op-system} (aarch64) image mirror]
 endif::ibm-power[]
 ifdef::ibm-power[]
 link:https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rhcos/[{op-system} image mirror]
 endif::ibm-power[]
-page.
+pages.
```

Comment:
Maybe move "page." and "pages." into their respective conditionals? Since they now differ singular vs plural.